manifests/instance/config.pp

```diff
@@ -208,6 +215,64 @@
   }
 
   ###
+  # Configure hostmanager & manager webapps 
+  #
+  
+  if !$tomcat::sources {
+    $webapps_base = $::osfamily ? {
+      'RedHat' => $::operatingsystemmajrelease ? {
+        '7'     => '/var/lib/tomcat/server/webapps',
+        default => "/var/lib/tomcat${tomcat::version}/server/webapps",
+      },
+      'Debian' => "/usr/share/tomcat${tomcat::version}-admin",
+    }
+  }else {
+    $webapps_base = '/opt/apache-tomcat/webapps'
+  }
+  
+      
+  if $manager {
+    file { "${catalina_base}/webapps/manager":
+      ensure => link,
+      target => "${webapps_base}/manager",
+    }
+  }elsif ($manage){
+    file { "${catalina_base}/webapps/manager":
+      ensure => absent,
+    }
+  }
```

Comment:
I would rather declare this file only once, and change the `ensure` parameter value depending on `$manager`.